Vagrantfile

```diff
@@ -30,14 +30,11 @@ Vagrant.configure("2") do |config|
   config.vm.network :private_network, ip: ENV.fetch("PFB_PRIVATE_IP", "192.168.111.111")
 
   config.vm.network :forwarded_port, guest: 9200, host: ENV.fetch("PFB_NGINX_PORT", 9200)
-  config.vm.network :forwarded_port, guest: 9210, host: ENV.fetch("PFB_GUNICORN_PORT", 9202)
-  config.vm.network :forwarded_port, guest: 9211, host: ENV.fetch("PFB_RUNSERVER_PORT", 9203)
-  config.vm.network :forwarded_port, guest: 9210, host: ENV.fetch("PFB_WEBPACK_PORT", 9210)
-  config.vm.network :forwarded_port, guest: 9211, host: ENV.fetch("PFB_WEBPACK_LR_PORT", 9211)
-  config.vm.network :forwarded_port, guest: 9212, host: ENV.fetch("PFB_WEBPACK_PROD_PORT", 9212)
-  config.vm.network :forwarded_port, guest: 9213, host: ENV.fetch("PFB_WEBPACK_PROD_LR_PORT", 9213)
+  config.vm.network :forwarded_port, guest: 9202, host: ENV.fetch("PFB_GUNICORN_PORT", 9202)
+  config.vm.network :forwarded_port, guest: 9203, host: ENV.fetch("PFB_RUNSERVER_PORT", 9203)
   config.vm.network :forwarded_port, guest: 5432, host: ENV.fetch("PFB_ANALYSIS_DB_PORT", 9214)
-
+  config.vm.network :forwarded_port, guest: 9301, host: ENV.fetch("PFB_REPOSITORY_PORT", 9301)
+  config.vm.network :forwarded_port, guest: 3001, host: ENV.fetch("PFB_REPOSITORY_BROWSERSYNC", 3001)
```

Comment:
Yes indeedy. Also needed to be fixed in the gulp server config.  I think gulp, docker-compose, Vagrant, and the README are now all on the same page...